wireshark_gen/templates/_ofclass_dissector.lua

```diff
@@ -43,7 +43,7 @@ function ${name}(reader, subtree)
 :: if ofclass.virtual:
     return ${ofclass.name}_v${version.wire_version}_dissectors[reader.peek(${ofclass.discriminator.offset},${ofclass.discriminator.length}):uint()](reader, subtree)
 :: else:
-:: if not ofclass.is_fixed_length:
+:: if not ofclass.is_fixed_length and ofclass.length_member is not None:
```

Comment:
I think this is caused by the the "7.3.5.19.1 Flow Monitoring Request", which is a multi-part request with this peculiar structure:
```
 * The OFPMP_FLOW_MONITOR request’s body consists of an array of zero or more
 * instances of this structure.
struct ofp_flow_monitor_request {
```

`ofp_flow_monitor_request` itself is variable length, because its member `ofp_match` is variable length; however, the outer structure only adds constant-length overhead. It seems that we don't deal with this situation in loxigen correctly right now (or @rlane, do you recall other instances where we have such an arrangement)?